docset/windows/dnsserver/Set-DnsServerDiagnostics.md

```diff
@@ -419,7 +419,15 @@ Accept wildcard characters: False
 
 ### -EventLogLevel
 Specifies an event log level.
-Valid values are Warning, Error, and None.
+Valid values are:
+
+0: No Events
```

Comment:
```suggestion
- 0: No Events
```

Reply:
I didn't use a markdown list on purpose, to match the style of the rest of the document, which contains 'lists' of options on other parameters, but does not user markdown lists. i.e. `DebugLogging` and `OperationalLogLevel1`. What would you like me to do, make the list styles inconsistent or 'apply markdown lists to the other lists too?

Reply:
It's true, I'm agree with you. Maybe it looks better but you choose.

Reply:
Agree with you.